Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
-    erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
       base64
       faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-http-cache (2.5.0)
+    faraday-http-cache (2.5.1)
       faraday (>= 0.8)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     faraday-net_http_persistent (2.1.0)
       faraday (~> 2.5)
       net-http-persistent (~> 4.0)
-    ffi (1.15.5)
+    ffi (1.16.3)
     ffi-compiler (1.0.1)
       ffi (>= 1.0.0)
       rake
-    flay (2.12.0)
-      erubis (~> 2.7.0)
+    flay (2.13.2)
+      erubi (~> 1.10)
       path_expander (~> 1.0)
       ruby_parser (~> 3.0)
       sexp_processor (~> 4.0)
     flowdock (0.7.1)
       httparty (~> 0.7)
       multi_json
-    forking_test_runner (1.2.0)
+    forking_test_runner (1.13.0)
       parallel_tests (>= 1.3.7)
-    git_diff_parser (3.1.0)
-    gitlab (4.10.0)
-      httparty (~> 0.14, >= 0.14.0)
-      terminal-table (~> 1.5, >= 1.5.1)
-    globalid (1.1.0)
-      activesupport (>= 5.0)
+    git_diff_parser (3.2.0)
+    gitlab (4.19.0)
+      httparty (~> 0.20)
+      terminal-table (>= 1.5.1)
+    globalid (1.2.1)
```

Comment:
### [globalid](http://www.rubyonrails.org)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.0 → 1.2.1

[_~~change-log~~, [source-code](https://github.com/rails/globalid), [gem-diff](https://my.diffend.io/gems/globalid/1.1.0/1.2.1)_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/rails/globalid/compare/v1.1.0...v1.2.1).

These are the first 10 commits:
- (9e59260) [Drop support to Rails < 5.2](https://github.com/rails/globalid/commit/9e5926080ccba4ba6e4ba667e2d4c58f9841720a)
- (f471fd5) [Drop support to all unsupported Ruby and Rails versions](https://github.com/rails/globalid/commit/f471fd5e62851d60d4d8c3ffbba25f7296ca4af7)
- (954ed95) [Merge pull request #153 from rails/rm-drop-unsupported-rails](https://github.com/rails/globalid/commit/954ed954363fc5ec5c8621992c359e7d3f69d312)
- (dc9a648) [Use ActiveSupport::MessageVerifier's expiry and purpose feat…](https://github.com/rails/globalid/commit/dc9a6487b3354ae2d2054b5fd5b4e57fbbad97bc)
- (bc6345f) [Add tests to make sure the new behavior of checking on the v…](https://github.com/rails/globalid/commit/bc6345fdf8ebc5c1371e10483b957045558ef438)
- (12f7629) [Default to new behavior but fallback parsing to old behavior](https://github.com/rails/globalid/commit/12f76297c803c658ea2fc684d4336db347a90901)
- (9d15bf2) [Merge PR #107](https://github.com/rails/globalid/commit/9d15bf2e931ac6ed4ba17936186657bd2001a2e4)
- (c620103) [Bump rack from 2.2.6.2 to 2.2.6.4](https://github.com/rails/globalid/commit/c6201039817bb8d8a5ba0f73338342381b41a5aa)
- (fe4249a) [Merge pull request #158 from rails/dependabot/bundler/rack-2…](https://github.com/rails/globalid/commit/fe4249a6418986a491616b244f5dddd1d0ff3e6b)
- (3b5ebe3) [Add RDoc documentation for `to_gid` and `to_sgid` methods](https://github.com/rails/globalid/commit/3b5ebe3782a9e8128597b5541eb565fc23bd33eb)


</details>